### PR TITLE
more lazier: auto copy to clipboard

### PR DESCRIPTION
**Pull request by Cirn09**

### CODE REVIEW

File: LazyIDA.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from __future__ import division
 from __future__ import print_function
 from struct import unpack
+from pyperclip import copy
```
Comment (L4ys) [inline LazyIDA.py]: We should just call `copy_to_clip` instead of using a external library here

File: LazyIDA.py [hunk 2]
```diff
@@ -171,24 +169,25 @@ def activate(self, ctx):
...
                     # python list dword
                     data += b"\x00" * 3
-                    print("[%s]" % ", ".join("0x%08X" % u32(data[i:i+4]) for i in range(0, size, 4)))
+                    output = "[%s]" % ", ".join("0x%08X" % u32(data[i:i+4]) for i in range(0, size, 4))
                 elif self.action == ACTION_CONVERT[9]:
                     # python list qword
                     data += b"\x00" * 7
-                    print("[%s]" %  ", ".join("%#018X" % u64(data[i:i+8]) for i in range(0, size, 8)).replace("0X", "0x"))
+                    output = "[%s]" %  ", ".join("%#018X" % u64(data[i:i+8]) for i in range(0, size, 8)).replace("0X", "0x")
+                copy(output)
```
Comment (L4ys) [inline LazyIDA.py]: We should just call `copy_to_clip` instead of using a external library here

File: README.md [hunk 3]
```diff
@@ -3,6 +3,7 @@ Make your IDA Lazy!
 
 # Install
 1. put `LazyIDA.py` into `plugins` folder under your IDA Pro installation path.
+2. `pip install pyperclip`
```
Comment (L4ys) [inline README.md]: This is unnecessary now

**Comment by Cirn09** [comment]
Ah, I just noticed that LazyIDA has clipboard-related features and functions...

**Pull request merged**